docs/source/en/using-diffusers/using_safetensors.mdx

```diff
@@ -1,87 +1,74 @@
-# Loaded in Pytorch 0:00:02.663379
-```
+For more explicit control, you can optionally set `use_safetensors=True` (if `safetensors` is not installed, you'll get an error message asking you to install it):
 
-This is for the entire loading time, the actual weights loading time to load 500MB:
+```py
+from diffusers import DiffusionPipeline
 
-```
-Safetensors: 3.4873ms
-PyTorch: 172.7537ms
+pipeline = DiffusionPipeline.from_pretrained("runwayml/stable-diffusion-v1-5", use_safetensors=True)
 ```
 
-Performance in general is a tricky business, and there are a few things to understand:
+However, model weights may not necessarily be stored in separate subfolders like in the example above. Sometimes, all the weights are stored in a single `.safetensors` file. In this case, load the file directly with the [`~diffusers.loaders.FromCkptMixin.from_ckpt`] method:
 
-- If you're using the model for the first time from the hub, you will have to download the weights.
-  That's extremely likely to be much slower than any loading method, therefore you will not see any difference
-- If you're loading the model for the first time (let's say after a reboot) then your machine will have to 
-  actually read the disk. It's likely to be as slow in both cases. Again the speed difference may not be as visible (this depends on hardware and the actual model).
-- The best performance benefit is when the model was already loaded previously on your computer and you're switching from one model to another. Your OS, is trying really hard not to read from disk, since this is slow, so it will keep the files around in RAM, making it loading again much faster. Since safetensors is doing zero-copy of the tensors, reloading will be faster than pytorch since it has at least once extra copy to do.
+```py
+from diffusers import StableDiffusionPipeline
 
-# How to use safetensors ?
+pipeline = StableDiffusionPipeline.from_ckpt(
+    "https://huggingface.co/WarriorMama777/OrangeMixs/blob/main/Models/AbyssOrangeMix/AbyssOrangeMix.safetensors"
+)
+```
 
-If you have `safetensors` installed, and all the weights are available in `safetensors` format, \
-then by default it will use that instead of the pytorch weights.
+## Convert to safetensors
 
-If you are really paranoid about this, the ultimate weapon would be disabling `torch.load`:
-```python
-import torch
+Not all weights on the Hub are available in the `.safetensors` format, and you may encounter weights stored as `.bin`. In this case, use the Spaces below to convert the weights to `.safetensors`. The Convert Spaces downloads the pickled weights, converts them, and opens a Pull Request to upload the newly converted `.safetensors` on the Hub. This way, if there is any malicious code contained in the pickled files, they're uploaded to the Hub - which has a [security scanner](https://huggingface.co/docs/hub/security-pickle#hubs-security-scanner) to detect unsafe files and suspicious pickle imports - instead of your computer.
```

Comment:
I'm always confused when I see plural `Spaces` being used for a (single) item in the Spaces hub 😅 Just a question for my understanding, is that the recommended way to refer to them?

Reply:
Haha good question. I use `Spaces` because it's the name of the product, kind of like "Docker container" or "Gradio app". The Hub [docs](https://huggingface.co/docs/hub/spaces) uses `Space` to describe a single Spaces, so I can adopt that here and moving forward :)